### PR TITLE
include-common-list-requests

### PR DESCRIPTION
**Pull request by Bezbran**
move identifiers to identifiers package
move all[ container scan repo](https://github.com/armosec/cluster-container-scanner-api) to container scan package here.
move notifications structure to dedicated package

### CODE REVIEW

File: armotypes/portaltypes.go [hunk 1]
```diff
@@ -53,124 +51,6 @@ type PortalBase struct {
...
-	WorkerNodeslastReported          = "lastReported"
-	WorkerNodeslastReportDate        = "lastReportDate"
-	WorkerNodesmaxPerMonth           = "maxPerMonth"
-	WorkerNodesmaxReportGUID         = "maxReportGUID"
-	WorkerNodesmaxPerMonthReportGUID = "maxPerMonthReportGUID"
-	WorkerNodeslastReportGUID        = "lastReportGUID"
-)
-
 // PortalCluster holds cluster data from portal BE
 type PortalCluster struct {
```
Comment (avrahams) [inline armotypes/portaltypes.go:55]: Shouldn't this move to the config service package?

File: armotypes/pagination_structs.go [hunk 2]
```diff
@@ -0,0 +1,180 @@
+package armotypes
```
Comment (avrahams) [inline armotypes/pagination_structs.go:1]: is armo types the right package ?
if we are already breaking it to packages these are not model types but API types

Comment (Bezbran) [reply armotypes/pagination_structs.go:1]: I am not sure about the division of types<>APIs, but I thought that APIs are for internal stuff, while armotypes is for external as well

File: armotypes/portaltypes.go (hunk 1 above)
Comment (Bezbran) [reply armotypes/portaltypes.go:55]: I moved only the customer struct since it's the only one which sufer from circular dependcies

**Review by avrahams — APPROVED** [review]

**Pull request merged**